src/tools/remove_surface.cc

```diff
@@ -59,7 +59,7 @@ struct remove_surface_tool : tool
         block *bl = rc.block;
         auto index = normal_to_surface_index(&rc);
 
-        auto &mesh = asset_man.get_surface_mesh(bl->surfs[index]);
+        auto &mesh = asset_man.surf_kinds.at(rc.block->surfs[index]).visual_mesh;
```

Comment:
rc.block?

Reply:
Equivalent. bl is now unused, which I should clean up.